share/completions/mdadm.fish

```diff
@@ -0,0 +1,206 @@
+# fish completions for mdadm, as documented under Debian Stretch (mdadm v3.4)
+# Known lacks :
+# - be more contextual : suggest according to available files, devices, arrays…
+# - especially, suggest values for levels, metadata format… according to the already given parameters
+# - some descriptions are not very useful, even a bit Captain Obvious-like
+# - some options could see their meaning in different contexts explained with different completions with proper tests for context
+# - misc and manage modes could be tested for, and the corresponding options proposed only in the right context
+#
+# -n tests have been simplified when __fish_contains_opt …; or _fish_not_contain_opt …: the options in the first test were removed from the second one,
+#       as their absence has already been checked, because this absence caused evaluation of __fish_not_contain_opt
+
+function __fish_mdadm_metadata_options
+	echo "0	Use original 0.90 format superblock"
```

Comment:
Or, a bit simpler:

```fish
echo "0"\t"Use original 0.90 format superblock"
echo "0.90"\t"..."
```

or countless variations. Anyway, please use `\t`, no tab characters in our files if we can help it.